tao_compiler/mlir/xla/ral/context/common_context_impl_quantization.cc

```diff
@@ -354,7 +354,7 @@ void ral_qconv_acl_s8_s8_s8_per_channel(
 ///        convolution creator.
 /// @param input [batch, in_height, in_width, in_channels] with "NHWC" or
 ///              [batch, in_channels, in_height, in_width] with "NCHW"
-/// @param weight [filter_height, filter_width, in_channels, out_channels]
+/// @param weight [out_channels, filter_height, filter_width, in_channels]
```

Comment:
please add a UT with input having different element values to make sure future change not break such requirement.